tests/unit/threads/CMakeLists.txt

```diff
@@ -59,6 +60,7 @@ set(thread_stacksize_PARAMETERS LOCALITIES 2)
 set(tss_PARAMETERS THREADS_PER_LOCALITY 4)
 
 ###############################################################################
+# TODO: put an option to add private or public to the libraries
```

Comment:
Not relevant for executables? Or `PUBLIC`/`PRIVATE` can just be set in `DEPENDENCIES` if needed.